shell/yq.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Wrapper around `yq` that allows use of python-yq or gojq.
+# You can force using gojq by setting the environment variable YQ_USE_GOJQ=true.
+#
+
```

Comment:
Probably want to use `set -euo pipefail` in all new shell files.

Reply:
Fair
```suggestion

set -euo pipefail
```